### PR TITLE
CIV-4219 Get the details right

### PR DESCRIPTION
**Pull request by hmcts-version1-domini**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-4219

### Change description ###

Develop the 'Get the details right' screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/unit/routes/features/claim/completingClaimController.test.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import nock from 'nock';
+import config from 'config';
+import {
+  CLAIM_COMPLETING_CLAIM_URL,
+} from '../../../../../main/routes/urls';
+import {TestMessages} from '../../../../utils/errorMessageTestConstants';
+import {mockCivilClaim} from '../../../../utils/mockDraftStore';
+
+jest.mock('../../../../../main/modules/oidc');
+jest.mock('../../../../../main/modules/draft-store');
```
Comment (commitBlob) [inline src/test/unit/routes/features/claim/completingClaimController.test.ts:12]: don't think you need draft-store mock

File: src/test/unit/routes/features/claim/completingClaimController.test.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  const idamUrl: string = config.get('idamUrl');
+  beforeEach(() => {
+    nock(idamUrl)
+      .post('/o/token')
+      .reply(200, {id_token: citizenRoleToken});
+  });
+
+  describe('on GET', () => {
+    it('should return completing claim page', async () => {
+      app.locals.draftStoreClient = mockCivilClaim;
```
Comment (commitBlob) [inline src/test/unit/routes/features/claim/completingClaimController.test.ts:25]: haven't seen in the controller you are using claim details. Don't think you need this either

File: src/test/unit/views/features/claim/completing-claim.test.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import request from 'supertest';
+import {app} from '../../../../../main/app';
+import {CLAIM_COMPLETING_CLAIM_URL} from '../../../../../main/routes/urls';
+import {mockCivilClaim} from '../../../../utils/mockDraftStore';
+
+const jsdom = require('jsdom');
+const {JSDOM} = jsdom;
+
+jest.mock('../../../../../main/modules/oidc');
+jest.mock('../../../../../main/modules/draft-store');
```
Comment (commitBlob) [inline src/test/unit/views/features/claim/completing-claim.test.ts:12]: don't think you need draft-store mock

File: src/test/utils/errorMessageTestConstants.ts [hunk 4]
```diff
@@ -41,6 +41,7 @@ export const enum TestMessages {
   VALID_YES_NO_SELECTION = 'Please select yes or no',
   VALID_DEFENDENT_AGE_ELIGIBILITY = 'Select yes, no, or company/organisation',
   MEDIATION_DISAGREEMENT = 'You chose not to try free mediation',
+  COMPLETING_CLAIM = 'Get the details right',
```
Comment (commitBlob) [inline src/test/utils/errorMessageTestConstants.ts:44]: is this error?

File: src/test/unit/routes/features/claim/completingClaimController.test.ts (hunk 1 above)
Comment (hmcts-version1-domini) [reply src/test/unit/routes/features/claim/completingClaimController.test.ts]: Right. Done. Thanks

File: src/test/unit/routes/features/claim/completingClaimController.test.ts (hunk 2 above)
Comment (hmcts-version1-domini) [reply src/test/unit/routes/features/claim/completingClaimController.test.ts]: Right. Done. Thanks

File: src/test/unit/views/features/claim/completing-claim.test.ts (hunk 3 above)
Comment (hmcts-version1-domini) [reply src/test/unit/views/features/claim/completing-claim.test.ts]: Right. Done. Thanks

File: src/test/utils/errorMessageTestConstants.ts (hunk 4 above)
Comment (hmcts-version1-domini) [reply src/test/utils/errorMessageTestConstants.ts]: Right, is not actually. Removed. Thanks

**Review by TatsyMaslova — APPROVED** [review]

**Review by commitBlob — APPROVED** [review]

**Review by GorilaNaranja — APPROVED** [review]

**Pull request merged**